core/vertx/pom.xml

```diff
@@ -29,6 +29,11 @@
       <artifactId>junit</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
test scope?

Reply:
Already is via dep-man:


      <dependency>
        <groupId>io.thorntail</groupId>
        <artifactId>thorntail-testing</artifactId>
        <version>4.0.0-SNAPSHOT</version>
        <scope>test</scope>
      </dependency>

Reply:
ignore me

Reply:
Hm, I wonder where to place the tests using `ThorntailTestRunner`? There is a dedicated test suite here:
https://github.com/thorntail/thorntail/tree/4.x/testsuite/simple/vertx

I think that `src/test/java` should only contain unit tests.